addons/mail/static/tests/discuss_app/sidebar.test.js

```diff
@@ -1317,10 +1317,10 @@ test("Sidebar compact is locally persistent (saved in local storage)", async ()
     await start();
     await openDiscuss();
     await contains(".o-mail-DiscussSidebar.o-compact");
-    await click("[aria-label='Make panel bigger']");
+    await click("[aria-label='Expand']");
```

Comment:
Since "collapse"/"expand" are less specific, I would put `.o-DiscussSidebar ` in selector. This reduces risk of clashing with thread actions in header or message actions, while clarifying better than the test expects to collapse / expand the discuss sidebar.